readme.md

```diff
@@ -285,12 +286,42 @@ as a demonstration.
 We have perfect accuracy and a spotless confusion matrix.  But of
 course, never trust evaluations on training data.
 
+### Clusters
+
+A [cluster](https://bigml.com/developers/clusters) is a set of groups
+of instances of a dataset that have been automatically classified
+together according to a distance measure computed using the fields of
+the dataset. Cluster are used in unsupervised learning.
```

Comment:
That line is a bit awkward.  Perhaps "A clustering is an example of unsupervised learning."?